src/corecel/io/ColorUtils.cc

```diff
@@ -44,11 +46,13 @@ bool use_color()
             // Color is explicitly enabled
             return true;
         }
+#ifndef _WIN32
```

Comment:
Is there a `Windows` equivalent to tell if there a terminal attached or not?

Reply:
I'm not sure at all... even if there is, the Windows terminals don't natively support ANSI color sequences (only when running through something like git-bash). Either way, for the use cases of concern to SCALE, we won't be running through the command line.

Reply:
humm ... isn't the code then
```
#ifdef _WIN32
     return false;
#else 
... existing code...
#endif
```

Reply:
You could be running through git-bash but compiled against only regular windows. (This is how I was testing.) In that case the ANSI sequences are processed correctly because git-bash defines the `TERM` variable.